tutor/utils.py

```diff
@@ -246,6 +246,17 @@ def check_output(*command: str) -> bytes:
         raise exceptions.TutorError(f"Command failed: {literal_command}") from e
 
 
+def get_compose_stacks() -> list[str]:
+    command = ("docker", "compose", "ls", "--format", "json")
+    try:
+        return [
+            entry["Name"]
+            for entry in json.loads(subprocess.check_output(command).decode("utf-8"))
```

Comment:
No need to call subprocess.check_output, you can already use the `check_output` command. And then we don't need to catch the exception, because check_output already takes care of that.